utils/iscsi-pr.c

```diff
@@ -0,0 +1,476 @@
+/* 
+   Copyright (C) 2022 by zhenwei pi<pizhenwei@bytedance.com>
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; either version 2 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#ifdef HAVE_POLL_H
+#include <poll.h>
+#endif
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <string.h>
+#include <getopt.h>
+#include <inttypes.h>
+#include "iscsi.h"
+#include "scsi-lowlevel.h"
+
+static const char *initiator = "iqn.2022.com.github:pizhenwei:libiscsi:iscsi-pr";
```

Comment:
Please do not hardcode iSCSI IQNs.

Reply:
It's possible to overwrite the default initiator by `-i, --initiator-name=iqn-name`, keep the same parameter with other iscsi utils. (Ex, iscsi-inq)